### PR TITLE
DOC: Add first interaction GHA workflow file

### PR DESCRIPTION
**Pull request by jhlegarreta**
- DOC: Add first interaction GHA workflow file
- DOC: Use sentence case for example commit message subject

### CODE REVIEW

**Comment by jhlegarreta** [comment]
See if this helps raising awareness about the guidelines and checks.

**Comment by arokem** [comment]
Great idea!

File: .github/workflows/first_interaction.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+          We would appreciate if you took the time to submit a pull request to fix this issue. Please read our
+          [CODE OF CONDUCT](../CODE_OF_CONDUCT.md) and our [CONTRIBUTING guidelines](../CONTRIBUTING.md) if you have not
+          done that already :book:.
+
+        pr-message: |
+          Thank you for contributing a pull request!
+
+          **We are glad that you are finding DIPY useful!**
+
+          This is an automatic message. Allow for time for DIPY maintainers to be able to read the issue and comment on
```
Comment (arokem) [inline .github/workflows/first_interaction.yml:35]:
```suggestion
          This is an automatic message. Allow for time for DIPY maintainers to be able to read this pull request and comment on
```

**Review by arokem — APPROVED** [review]
Small nit

File: .github/workflows/first_interaction.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+    steps:
+    - uses: actions/first-interaction@v1
+      with:
+        repo-token: "${{ secrets.GITHUB_TOKEN }}"
+        issue-message: |
+          Thank you for contributing an issue!
+
+          **We are glad that you are finding DIPY useful!**
+
```
Comment (skoudoro) [inline .github/workflows/first_interaction.yml:22]: I would add a sentence saying to move the issue to discussions if asking for help or advice. issues a dedicated for new features request or bug founded.

File: .github/workflows/first_interaction.yml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+          This is an automatic message. Allow for time for DIPY maintainers to be able to read the issue and comment on
+          it.
+
+          Note that we require the **code formatting**, **testing** and **documentation builds** to **pass** in order to
+          merge your pull request. GitHub will report on their status as the builds become available. **Please, check
+          their status and make the appropriate changes as necessary**. It is your responsibility to do this to have
+          your pull request merged :mag:.
```
Comment (skoudoro) [inline .github/workflows/first_interaction.yml:41]: `to have your pull request reviewed on time and merged`

**Review by skoudoro — COMMENTED** [review]
I agree that it will be helpful. Thanks @jhlegarreta.

See below few comments

File: .github/workflows/first_interaction.yml (hunk 2 above)
Comment (jhlegarreta) [reply .github/workflows/first_interaction.yml:22]: Done https://github.com/dipy/dipy/pull/3387/commits/ab12ade3cb0ce2d44b4827145d14c507be853a10#diff-0389854d99ec31d393a8a889246f3b9aa55395e692bbfa2a3142b8f28adf5ed0R26.

File: .github/workflows/first_interaction.yml (hunk 3 above)
Comment (jhlegarreta) [reply .github/workflows/first_interaction.yml]: Reworded. Resolving.

**Review by skoudoro — APPROVED** [review]
All good, ok for merging from my side.

Thanks @jhlegarreta!

@arokem?

**Pull request merged**